lib/mountain_view/presenter.rb

```diff
@@ -0,0 +1,70 @@
+module MountainView
+  class Presenter
+    class_attribute :_attributes, instance_accessor: false
+    self._attributes = {}
+
+    attr_reader :slug, :properties
+    alias_method :props, :properties
```

Comment:
Ah, I see! I think I'd be fine to remove it, and people can always alias in in their inherited presenters if they want.